content/api/commands/mount.md

```diff
@@ -589,6 +514,172 @@ it('login link should be active when url is "/login"', () => {
 </template>
 </code-group-vue2-vue3>
 
+### Vuex
+
+To use a component that uses [Vuex](https://vuex.vuejs.org/), you can create a
+`mountWithVuex` command that expose a Vuex store to your component:
+
+<code-group-vue2-vue3>
+<template #vue2>
+
+```js
+import { mount } from '@cypress/vue'
+import Vuex from 'vuex'
+import { getStore } from '../../src/plugins/store'
+
+Cypress.Commands.add('mountWithVuex', (comp, options = {}) => {
+  // Setup options object
+  options.extensions = options.extensions || {}
+  options.extensions.plugins = options.extensions.plugins || []
+
+  // Use store passed in from options, or initialize a new one
+  options.store = options.store || getStore()
+
+  // Add Vuex plugin
+  options.extensions.plugins.push(Vuex)
+
+  return mount(comp, options)
+})
+```
+
+> The `getStore` method is a factory method that initializes Vuex and creates a
+> new store. It is important that the store be initialized with each new test to
+> ensure changes to the store don't affect other tests.
+
+Typings:
+
+```ts
+import { mount } from '@cypress/vue'
+import { Store } from 'vuex'
+
+type MountParams = Parameters<typeof mount>
+type OptionsParam = MountParams[1]
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Helper mount function for Vue Components
+       * @param component Vue Component or JSX Element to mount
+       * @param options Options passed to Vue Test Utils
+       */
+      mountWithVuex(
+        component: any,
+        options?: OptionsParam & { store?: Store }
+      ): Chainable<any>
+    }
+  }
+}
+```
+
+Usage:
+
+```js
+import { getStore } from '@/plugins/store'
+import UserProfile from './UserProfile.vue'
+
+it.only('User profile should display user name', () => {
+  const user = { name: 'test person' }
+
+  // getStore is a factory method that creates a new store
+  const store = getStore()
+
+  // mutate the store with user
+  store.commit('setUser', user)
+
+  cy.mountWithVuex(UserProfile, {
+    store,
+  })
+
+  cy.get('div.name').should('have.text', user.name)
+})
+```
+
+</template>
+<template #vue3>
+
+```js
+import { mount } from '@cypress/vue'
+import { getStore } from '../../src/plugins/store'
+
+Cypress.Commands.add('mountWithVuex', (comp, options = {}) => {
+  // Setup options object
+  options.global = options.global || {}
+  options.global.stubs = options.global.stubs || {}
+  options.global.stubs['transition'] = false
+  options.global.components = options.global.components || {}
+  options.global.plugins = options.global.plugins || []
+
+  // Use store passed in from options, or initialize a new one
+  const { store = getStore(), ...mountOptions } = options
+
+  // Add Vuex plugin
+  options.global.plugins.push({
+    install(app) {
+      app.use(store)
+    },
+  })
+
+  return mount(comp, mountOptions)
+})
+```
+
+> The `getStore` method is a factory method that initializes Vuex and creates a
```

Comment:
Instead of `>` use an `<Alert type="info"></Alert>`

Reply:
done in 7888195